chart/operator/templates/k8sgpt-rbac.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "chart.fullname" . }}-operator-role
```

Comment:
I think we need a better naming to designate that this is for K8sGPT workload and not operator related